content/news/playtest-20150322.md

```diff
@@ -0,0 +1,38 @@
+The next release is drawing closer as we enter the feature freeze phase of the development cycle with this new playtest.
+
+You can help our efforts to create a stable and balanced release by testing and [reporting bugs](http://bugs.openra.net) or [giving feedback](/community/) on our various social media sites and forums, or simply in the comments section below.
+
+Since the last playtest, we have continued our focus on the Dune 2000 mod, which has received a brand new sidebar. The sandworms feature has been polished, and harvester insurance has been added, so if you lose your only harvester to a sandworm, it will be replaced automatically, for free!
```

Comment:
I hope not. I'd be against releasing a playtest without this. All the alternatives suck (Merging this after the next playtest and doing another one, merging this after the playtest without doing another one, not merging this at all before the release).